Makefile

```diff
@@ -1,23 +1,27 @@
 SHELL=/bin/bash
 
-all: lint generate
+# Run `make schema` when updating this version and commit the created files.
+# TODO(lyarwood) - Host the expanded JSON schema elsewhere under the kubevirt namespace
+export KUBEVIRT_VERSION = main
```

Comment:
is this variable used somewhere?

Reply:
Yes both the schema and validate scripts use it.

```
$ git grep -l KUBEVIRT_VERSION scripts/
scripts/schema.sh
scripts/validate.sh
```